rllib/utils/metrics/stats.py

```diff
@@ -228,10 +220,14 @@ def __init__(
         # previous `reduce()` result in hist[1].
         self._hist = deque([0, 0, 0], maxlen=3)
 
-        self._throughput = throughput if throughput is not True else 0.0
-        if self._throughput is not False:
-            assert self._reduce_method == "sum"
-            assert self._window in [None, float("inf")]
+        self._throughput = 0.0
+        self._measure_throughput = False
+        if (
+            self._reduce_method == "sum"
```

Comment:
Dumb question: Why is this only relevant for `sum`? Because these are the only lifetime stats?

Reply:
This is the only reduce setting, for which it makes sense to measure throughput for. Doesn't really make sense for min/max/mean.